### DISCUSSION TITLE
Need help with combobox search focus issue

### DISCUSSION THREAD

**Discussion by Sajarin-M** [mantinedev/mantine#5186] [Q&A] [answered]
I'm trying to use a combobox example from the Mantine V7 website with a search feature in the dropdown. However, there's a problem: when I type in the search box and press the "Tab" key, instead of moving to the next input on the page, it takes me to the browser's address bar. Can someone please explain why this is happening? Am I doing something wrong?

You can see the issue in this [CodeSandbox](https://codesandbox.io/s/mantine-combobox-focus-issue-g2ftv5) demo. For a clearer view, open it in a new browser window.

[Combobox.webm](https://github.com/mantinedev/mantine/assets/68892162/092171eb-a8ae-4f6c-a9d6-2fbac3b6fd6c)

**Comment by rtivital** [comment]
In your sandbox focus behaves as expected, it moves focus to the next input

**Reply by Sajarin-M** [reply]
I apologize for the error in the example. It occurs when 'withinPortal' is set to 'true' on the combobox.
I have updated the [sandbox](https://codesandbox.io/s/mantine-combobox-focus-issue-g2ftv5?file=/src/App.jsx).
Is this how portals are supposed to work? I'm not very familiar with how portals work

**Reply by rtivital** [reply, marked as answer]
Yes, that is how portals work, they render context outside the current DOM tree. Mantine Portal component appends these DOM nodes before the closing body tag.

**Reply by Sajarin-M** [reply]
Thanks
